src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookTests.java

```diff
@@ -29,4 +36,47 @@ public void Facebook_001_removeWikiaApps() {
     settingsFB.openApps();
     settingsFB.removeAppIfPresent();
   }
+
+  @Test(groups = {"Facebook_002", "Facebook"})
+  public void Facebook_002_noEmailPerms() {
+    WikiBasePageObject base = new WikiBasePageObject(driver);
+    FacebookMainPageObject fbLogin = base.openFacebookMainPage();
+    FacebookUserPageObject userFB;
+    userFB = fbLogin.login(credentials.emailFB, credentials.passwordFB);
+    userFB.verifyPageLogo();
+    SignUpPageObject signUp = userFB.openSpecialSignUpPage(wikiURL);
+    FacebookSignupModalComponentObject fbModal = signUp.clickFacebookSignUp();
+    fbModal.acceptWikiaAppPolicyNoEmail();
+    String userName = "QA" + signUp.getTimeStamp();
+    String password = "Pass" + signUp.getTimeStamp();
+    String email = credentials.emailQaart2;
+    String emailPassword = credentials.emailPasswordQaart2;
+    MailFunctions.deleteAllEmails(email, emailPassword);
+    fbModal.typeUserName(userName);
+    fbModal.typePassword(password);
+    fbModal.typeEmail(email);
+    fbModal.createAccount();
+    signUp.verifyUserLoggedIn(userName);
+    signUp.verifyEmailNotConfirmed();
+    try {
+      Thread.sleep(5000);
```

Comment:
I used sleep because I wanted to give time for the email to arrive in the inbox, as I was having timing issues with trying to find the email before it arrived.  If you know a better solution I am happy to hear it.